src/scripts/public/single.js

```diff
@@ -0,0 +1,41 @@
+import { qs, addSafeEventListener } from './utils';
+
+/**
+ * Single post manipulation.
+ * [@return](https://github.com/return) {void}
+ */
+const bcgovBlockThemeSinglePost = () => {
+    /*
+     * SafarIE iOS requires window.requestAnimationFrame update.
+     */
+    window.requestAnimationFrame( () => {
+        /**
+         * Check if this is a single post.
+         */
+        const body = qs( '.single-post' );
+        if ( body ) {
+            /**
+             * Get the header and content elements, make sure they exist, calculate height of header, and subtract it from content min height.
+             */
+            const headerGroup = qs( '.bcgov-header-container' );
+            const content = qs( '.bcgov-body-content' );
+            if ( headerGroup && content ) {
+                const headerGroupHeight = window
+                    .getComputedStyle( headerGroup )
+                    .getPropertyValue( 'height' );
+                content.style.minHeight =
```

Comment:
this can go here https://github.com/bcgov/bcgov-wordpress-block-theme/blob/development/src/scripts/public/dom-loader.js#L58 and update it from content -> postContent, and remove this js file